### ISSUE TITLE
Movement Config uses the FacePosX/Y/Z when using API

### ISSUE THREAD

**Issue by TheArtGun** [DenchiSoft/VTubeStudio#34]
While normally the config can be adjusted specifically with how much the user wants it to activate, the API can't be fine tuned separate from the HeadPosX/Y/Z values- this both leads to jitter (as the raw input cannot be smoothed) as well as being unable to control how the model moves.

A solution would be to allow people to change what the movement config inputs are, almost like the parameter boxes but for the movement config.
This would open up the possibility of using hand location (for example) to control the movement config, for a sock puppet like effect, or any number of other controls (including custom ones from plugins)!
A smoothing control would be beneficial, as base values can get quite jittery and there's no way to help it at the moment.

Overall-
In addition to position, a rotation added to the config might be fun, (thinking about the sock puppet), giving users a greater degree of control beyond the base rigging of their models. 

In summary-
Add input/smoothing options to the movement config, as well as rotation (could put the current mouse wheel control as an additive modifier on the rotation input so you still get the overall rotation control)

Super rough mockup below-

![image](https://user-images.githubusercontent.com/46326918/166118382-463561ee-ebf1-4fa2-9f4d-cc80caed7c3d.png)
